src/selectors/hosts.ts

```diff
@@ -11,3 +11,19 @@ export const getHostTableRows = createSelector(
   getHosts,
   hosts => hosts.map(host => Object.values(host))
 );
+
+const hostsValuesSelector = state => state.hosts.items;
```

Comment:
Please check the changes which PR#34 introduces. It defines the actual structure of the host. So you can match the selectors to it to save time reimplementing the selectors later.
https://github.com/metalkube/facet/pull/34/files#diff-d75362b21270945abccac41ae9de67c5

Reply:
Your selector should be based on hosts data from `getHosts`.  hostsValuesSelector should not be needed.